_posts/release/2022-12-01-accumulo-2.1.1.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Apache Accumulo 2.1.1
+sortableversion: '02.01.01'
+LTM: true
+draft: true
+---
+
+** DRAFT RELEASE NOTES **
+
+## About
+
+Apache Accumulo 2.1.1 is a patch release of the 2.1.0 LTM line. 
+
+### Major Improvements
+
+### Minor Improvements
+
+* {% ghi 3077 %} Call FSDataOutputStream.setDropBehind when reading and writing WAL files
+* {% ghi 3079 %} Call FSDataOutputStream.setDropBehind on major compaction input files
+* {% ghi 3023 %} Reduced the amount of logging when an ExternalCompaction is cancelled for cause
+
+### Behavior Changes
+
+* {% ghi 3082 %} ScanServers can now scan offline tables. The AccumuloClient will no longer
+  throw a TableOfflineException when creating a Scanner or BatchScanner against an offline table.
+  Instead, an error will be thrown on the call to iterator().
+
+### Bug Fixes
+
+* {% ghi 3059 %} Remove stale compactions from CompactionCoordinator internal structure
+* {% ghi 3057 %} Modified Monitor to direct DataTable errors to console
```

Comment:
I don't think this one was a bug either... just a very slightly improved user experience with the monitor under some edge cases when the cluster isn't available. It might not even be worth bothering to mention, except the list of things for this patch release is pretty small, so we're not really risking losing the reader's attention by including it, I suppose.

Reply:
Moved to Minor Improvements in 0a951b3.